src/css/main.module.less

```diff
@@ -228,9 +228,21 @@
 		}
 	}
 
+	h2 {
+		margin-top: 2rem;
+	}
```

Comment:
Would you please tell me any place where I can check this change affected?

Reply:
You can see the examples of changed font sizes in this page.
https://enactjs.com/docs/tutorials/tutorial-hello-enact/basics/
- h1: Enact Basics (title)
- h2: Building the App module, Rendering the App
- h3: The Module’s `package.json`, The Module’s Source File
- h4: App component, Exporting the App
- h5: `const` vs `let`, Composing Components